### DISCUSSION TITLE
Reverse conversion - mp3/wav/etc. to AAX

### DISCUSSION THREAD

**Discussion by biomichael** [audiamus/BookLibConnect#155] [Ideas]
I'm interested in a converter from any audio file to AAX for sideloading to an e-ink device. So far I have not seen anything that would do it, with and without Amazon account. I'm a developer for a not for profit and would like to have an ability to quality check my audio files on a final device.

**Comment by audiamus** [comment]
.aax is not a real file format. It's Mpeg4, audio only (.m4b, .m4a), with an encrypted AAC audio stream. You could simply rename .m4b to .aax and see how your target app handles it.
